src/lib/auto-updater.js

```diff
@@ -29,16 +52,33 @@ function setup () {
   })
 }
 
-export default async function () {
+async function checkForUpdates () {
+  try {
+    await autoUpdater.checkForUpdates()
+  } catch (_) {
+    // Ignore. The errors are already handled on 'error' event.
+  }
+}
+
+export default async function (ctx) {
   if (process.env.NODE_ENV === 'development') {
+    ctx.checkForUpdates = () => {
+      notify({
+        title: 'DEV Check for Updates',
+        body: 'Yes, you called this function successfully.'
+      })
+    }
+
     return
   }
 
   setup()
 
-  try {
-    await autoUpdater.checkForUpdates()
-  } catch (_) {
-    // Ignore. The errors are already handled on 'error' event.
+  await checkForUpdates()
+  setInterval(checkForUpdates, 3600000) // every hour
```

Comment:
I believe pinging servers every hour is too often, especially now that we add an option to do a manual check.

Firefox checks every 43200 seconds (12 hours):
```suggestion
  setInterval(checkForUpdates, 43200000) // every 12 hours
```